opentreemap/treemap/js/src/lib/utility.js

```diff
@@ -129,6 +129,14 @@ exports.offsetLatLngByMeters = function(latLng, dx, dy) {
     return { lat: newLat, lng: newLng };
 };
 
+// TODO: Respect instance units configuration.
```

Comment:
We already had https://github.com/OpenTreeMap/otm-addons/issues/326 for modeling -- I noted that it also applies to GSI polygons, and added a link in the comment.